contracts/lib/MetaBlock.sol

```diff
@@ -120,6 +125,19 @@ library MetaBlock {
          */
         bytes20 coreIdentifier;
     }
+    /**
+     * Seal object which tracks vote of validators for a given transition hash.
+     */
+    struct Seal {
+        /**
+         * Mapping of validator address and boolean which represents all the
+         * validators who has voted for given transition object.
+         */
```

Comment:
This comment is misleading. You describe how you use this field in the implementation of `OriginCore`. But if you abstract it and see it only in the context of this library, which should be re-usable across contracts, what you do with this mapping is "track which validators have been added to this seal". Right?